kappa_max: 0.2 # [1/m]
sigma_max: 0.2 # [1/m^2]

footprint:
  [[1.939181710715502, -1.0286977571988751],
  [-0.3230526413866528, -0.9273312536014424],
  [-0.7551751251156134, -0.8387417946752345],
  [-0.9158416333449395, -0.7538806793332407],
  [-1.0128531176987394, -0.6182469411020912],
  [-1.075682557253515, -0.3415106751077752],
  [-1.1019999999999754, 3.132672855543639e-05],
  [-1.075685196152108, 0.34155032793523277],
  [-1.0128541928055483, 0.6182679790131778],
  [-0.9158401672901277, 0.7538925559766716],
  [-0.7551695541073968, 0.8387480186215106],
  [-0.32303602609903237, 0.9273315324697313],
  [1.9084053111936896, 1.0573022428011376],
  [3.300487961089061, 0.9170112669142958],
  [3.5645888374441483, 0.7865653401846038],
  [3.7102144054238124, 0.6142783443781303],
  [3.796602563558224, 0.45675872738677925],
  [3.8240000000000123, 3.142418881396658e-05],
  [3.7964800013778017, -0.4567267739226446],
  [3.710094091194037, -0.6142570140860977],
  [3.5644721394828593, -0.7865556076677132],
  [3.300378104716742, -0.917010305823913]]

wheel_base:   2.912  # [m]
track_width:  1.584  # [m]
wheel_radius: 0.339  # [m]
wheel_width:  0.245  # [m]

motion_noise: {alpha1: 0.1,  alpha2: 0.05,
               alpha3: 0.05, alpha4: 0.2}

measurement_noise:
  std_x:     0.1     # [m]
  std_y:     0.1     # [m]
  std_theta: 0.05    # [rad]

controller: {k1: 1.5, k2: 0.25, k3: 1.0}
